### ISSUE TITLE
Copy/Paste Functionality

### ISSUE THREAD

**Issue by NetizenAbel** [DealPete/Sputnik#27]
**Describe the current limitation**
As it stands, the app does not allow for highlighting and copying text. The entire Edit menu is not yet present.

**Describe the proposed feature or enhancement**
Just as with essentially every app, a simple edit menu and attending modifications should do.

**Comment by diepoe** [comment]
Yeah, I also recognized that, I could work on this

**Comment by DealPete** [comment]
This is definitely needed.

**Comment by NetizenAbel** [comment]
Upon starting to work on this, I realized something. There isn't even a way to highlight text in the first place! I'm not sure where to start for that.

**Comment by DealPete** [comment]
Yeah, back when I wrote this, I remember there being no way to highlight text in SwiftUI. I was hoping that this was something they had added since; otherwise we'll just have to leave it as is.

**Comment by diepoe** [comment]
Yeah, there seems to be some "Hack" where you can use a TextField for that but that didn't looked well...

**Comment by diepoe** [comment]
I don't know if this could work, but I worked in another SwiftUI App with a WebKit-Renderer, where you can select/copy text from. Maybe we could coud render the sites with WebKit to resolve this issue? 🤔

**Comment by ghost** [comment]
Yeah, I suppose we could do that... though Gemini is supposed to be an alternative to the web. It would feel a little contrary to the spirit of things to render parts of the application in a web browser.

**Comment by diepoe** [comment]
Yeah, totally agree. That is the point I'm struggeling accepting this solution 👍

**Comment by adiabatic** [comment]
I use Elaho on iPadOS and iOS. In a nutshell, it uses Firefox but seems to generate a webpage and just displays that. Elaho also adds an Up button in the toolbar because, well, it has the space and is useful.

Elaho is a _very_ good client because it is able to leverage the gazillions of engineer-hours that went into making a browser front-end (setting up tabs and bookmarks, obviously; it's using WebKit anyway). Code re-use is OK.
